README.rst

```diff
@@ -94,7 +94,7 @@ Options
 -S FILE, --state FILE           Store program state (e.g., timestamps before
                                 which all asset are known to have been fetched)
                                 in the given file [default value:
-                                ``.dlstate.json``]
+                                ``.tinuous.json``]
```

Comment:
if we are renaming / breaking things up, may be we could harmonize it a bit more right away? e.g. have
- `tinuous.yml`
- `.tinuous-state.yml`
not that anyone should look into the `-state` one but still IMHO nice to have things consistent and descriptive, or otherwise I will find myself looking into .tinuous.json for configuration ;)

also note that https://yaml.org/faq.html recommends `.yaml` over `.yml` and thus we use it for `dandiset.yaml`.